lib/Marlin/Marlin/src/inc/Conditionals_LCD.h

```diff
@@ -569,6 +569,36 @@
 #undef EXTENDABLE_EMU_MMU2
 #undef EXTENDABLE_EMU_MMU2S
 
+/**
+ *  Multi-Material-Unit supported models
+ */
+#define PRUSA_MMU1             1
+#define PRUSA_MMU2             2
+#define PRUSA_MMU2S            3
+#define EXTENDABLE_EMU_MMU2   12
+#define EXTENDABLE_EMU_MMU2S  13
+
+#ifdef MMU_MODEL
+  #define HAS_MMU 1
+  #if MMU_MODEL == PRUSA_MMU1
+    #define HAS_PRUSA_MMU1 1
+  #elif MMU_MODEL % 10 == PRUSA_MMU2
+    #define HAS_PRUSA_MMU2 1
+  #elif MMU_MODEL % 10 == PRUSA_MMU2S
+    #define HAS_PRUSA_MMU2 1
+    #define HAS_PRUSA_MMU2S 1
+  #endif
+  #if MMU_MODEL >= EXTENDABLE_EMU_MMU2
+    #define HAS_EXTENDABLE_MMU 1
+  #endif
+#endif
+
+#undef PRUSA_MMU1
+#undef PRUSA_MMU2
+#undef PRUSA_MMU2S
+#undef EXTENDABLE_EMU_MMU2
+#undef EXTENDABLE_EMU_MMU2S
+
```

Comment:
Yuri explained, that it comes from later marlin 2.1.x branch than my update marlin was based on.

Reply:
Past versions required to define HAS_PRUSA_*, but in the new Configuration.h the user is expected to define the MMU model type. Note that private has already this change to that we can keep upstream Conditionals_LCD, although we replace the actual implementation.